pkg/kritis/metadata/metadata.go

```diff
@@ -24,17 +24,17 @@ import (
 
 type Fetcher interface {
 	// Vulnerabilities returns package vulnerabilities for a given image.
-	Vulnerabilities(containerImage string) ([]Vulnerability, error)
+	Vulnerabilities(containerImage string, auths []kritisv1beta1.AttestationAuthority) ([]Vulnerability, error)
```

Comment:
Like a couple of other comments suggest, no need for this change, because the project where the vulnerability occurrences live is in fact determined by containerImage.

Reply:
Hi! I think there won't be any problems addressing the other comments, but we might for this one.
Infact, in this PR we have this in `fetchOccurrence` (which is invoked by both the `Attestations` and `Vulnerabilities` functions):

```
...
req := &grafeas.ListNoteOccurrencesRequest{
		Name: auth.Spec.NoteReference,
...
```

So we need to pass an `AttestationAuthority` parameter to `fetchOccurrence`, as it is needed to specify the `noteReference`, which is required for the logic of `Attestations` (https://github.com/grafeas/kritis/pull/399/files#diff-a4c6acceb6d3cac8b345f5b3dfd0f93aR71) to provide for the separation of projects.

How do you think we should address this?

Reply:
Let's for now have separate methods for fetching occurrences for GAP and ISP. We can worry about sharing the logic later, in a subsequent PR, but writing something that'd share the code for the common bits would be great.

Reply:
Yes, I agree, having one method to fetch vulnz occurrences, and another one to fetch attestations, is good at least for now.  Thanks!

Reply:
Sounds good, thanks!

Reply:
Hi @aysylu and @nenaddedic, I've pushed a couple of new commits to address your comments.
Please let me know if this is now ready to get merged.